pkg/stanza/operator/input/windows/xml.go

```diff
@@ -118,9 +120,21 @@ func (e *EventXML) parseBody() map[string]interface{} {
 		"keywords":    keywords,
 		"event_data":  parseEventData(e.EventData),
 	}
+
 	if len(details) > 0 {
 		body["details"] = details
 	}
+
+	if e.Security != nil && e.Security.UserID != "" {
+		body["security"] = map[string]any{
```

Comment:
nit: other locations on this file use `map[string]interface{}` 

@djaglowski what is the general preference on this type of nit issue? Consistency, brevity, latest language patterns? (I have my preferences, but, the biggest one is following the practices of the repo/package).

Anyway, this is not material no need to hold merge because of it.

Reply:
Personally I'd prefer to take steps towards latest language patterns but I don't feel strongly about it here.